metaflow/plugins/argo/argo_workflows.py

```diff
@@ -189,6 +230,51 @@ def trigger_explanation(self):
                 "This workflow triggers automatically via the CronWorkflow *%s*."
                 % self.name
             )
+        elif len(self._triggering_flows) > 0 or len(self._triggering_events) > 0:
```

Comment:
it seems from this block that `@schedule` and `@trigger...` both are not supported concurrently, but I don't see that enforced anywhere in the code.